third_party/terraform/resources/resource_sql_database_instance.go

```diff
@@ -325,6 +325,13 @@ func resourceSqlDatabaseInstance() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"root_password": {
```

Comment:
Since SQL server in GCP is still in beta, I think we should put this property only in the beta provider. If you agree, then you'll want to change the file extension to `.go.erb` and add tags like the ones  you'll find in https://github.com/GoogleCloudPlatform/magic-modules/blob/master/third_party/terraform/resources/resource_compute_network_peering.go.erb for example. Same thing applies for the test, and then you'll want to say that it's beta in the docs

Reply:
yes, sure

Reply:
@danawillow  I guess it means also mark `root_password`  as beta in usage doc?